internal/utils/utils.go

```diff
@@ -86,7 +86,7 @@ func FindMatches(pattern string, text []byte) map[string]interface{} {
 	r := regexp.MustCompile(pattern)
 	values := r.FindAllStringSubmatch(string(text), -1)
 	for _, match := range values {
-		key := strings.Replace(match[1], " ", "_", -1) //nolint
+		key := strings.Replace(strings.TrimSpace(match[1]), " ", "_", -1) //nolint
```

Comment:
I don't really get this change. If we have spaces in our match, it means that the pattern we provided is wrong, and we should fix that, instead of changing the functionality here

Reply:
I think this patter should make the job:
```
(?m)^(\S+(?: \S+)*)\s*:\s(\S*)$
```
(It allows words and 2 words with a unique space in between)
https://regex101.com/r/OCmlXk/1